src/@utils/accessDetailsAndPricing.ts

```diff
@@ -222,13 +222,21 @@ function getAccessDetailsFromTokenPrice(
   return accessDetails
 }
 
+function getValidUntilTime() {
+  const mytime = new Date()
+  mytime.setMinutes(mytime.getMinutes() + 19)
+  return Math.floor(mytime.getTime() / 1000)
+}
+
 /**
  * This will be used to get price including feed before ordering
  * @param {AssetExtended} asset
  * @return {Promise<OrdePriceAndFee>}
  */
 export async function getOrderPriceAndFees(
   asset: AssetExtended,
+  computeEnv: string = null,
```

Comment:
put compute env after accountId , because we use accountId more often so we don't need to add `null, null` in params

Reply:
done my bad :) saw that `accountId?` was optional and didn't  want the linter and build to fail because we have optional values before required params, and for some reasons i forgot that initialised method params are considered optionals by default 🤦